File: src/main/java/com/likelionknu/applyserver/mail/data/dto/MailRequestDto.java
package com.likelionknu.applyserver.mail.data.dto;

import com.likelionknu.applyserver.auth.data.entity.User;
import com.likelionknu.applyserver.mail.data.entity.MailContent;
import lombok.Builder;
import lombok.Data;
import java.util.List;

@Data
@Builder
public class MailRequestDto {
    private User user;
    private String email;
    private String title;
    private String template;
    private List<MailContent> dataList;
}

File: src/main/kotlin/com/likelionknu/applyserver/mail/data/dto/MailRequestDto.kt
package com.likelionknu.applyserver.mail.data.dto

import com.likelionknu.applyserver.auth.data.entity.User
import com.likelionknu.applyserver.mail.data.entity.MailContent

data class MailRequestDto (
        val user: User? = null,
        val email: String,
        val title: String,
        val template: String,
        val dataList: List<MailContent>
)